lib/connection-manager.js

```diff
@@ -4,6 +4,7 @@ var debug = require('debug')('logdna:lib:connection-manager');
 var log = require('./log');
 var spawn = require('child_process').spawn;
 var os = require('os');
+var async = require('async');
```

Comment:
This may sound ridiculous, but alphabetically ordering `require`s within groups is helpful for maintenance.